src/components/modals/CreateAntModal/CreateAntModal.tsx

```diff
@@ -258,7 +258,7 @@ function CreateAntModal() {
           <CloseIcon width={30} height={30} fill={'var(--text-white'} />
         </button>
         <Workflow
-          stage={workflowStage}
+          stage={workflowStage.toString()}
```

Comment:
why do we need `toString()` here?

Reply:
changed the stage prop to a string type instead of number, it was complaining about the number - the workflow will be updated later though/